### DISCUSSION TITLE
touche screen

### DISCUSSION THREAD

**Discussion by youkorr** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#35] [General]
Good evening I would like to know for everyone what an esp32s3 box 3 if you had a problem when flashing your esp the touch screen does not respond because on my side I have to do a reboot of the box for it to work

**Comment by BigBobbas** [comment]
This is something that I was experiencing originally , but have not had the issue now for a number of weeks.  I assumed that this was solved in a recent ESPHome update, but I have since found out that someone else is also having this issue.  I do not know what is causing this as I am not experiencing this issue. As soon as I know of a fix I will post it.
Cheers
Rob

**Comment by youkorr** [comment]
ok I am reassured I thought my box was faulty

**Comment by Bcukier** [comment]
fwiw, I experience this every time as well.  A simple reset fixes, but the first time the box3 comes online after an install, the touch screen is reported as disabled.
